perf/README

```diff
@@ -131,3 +131,9 @@ call. Use option -k to select key type for benchmark.  The list of keys for
 testing is as follows: dh, dhx, dsa, ec, rsa, xkey.  To run bencmark for all
 keys and formats using 4 threads run pkeyread as follows:
     ./pkeyread -f all -k all -t 4
+
+run-perf.sh
+-----------
+
+Script which collects performance stats by running all tools above. It tests
+OpenSSL versions 1.1.1 3.0 and master.
```

Comment:
What about 3.1, 3.2, 3.3? Aren't we interested in those too? In particular we should be interested in master vs the latest released version (3.3) - so perhaps 1.1.1, 3.0, 3.3 and master are sufficient.

Reply:
OK, so I will add 3.3 to the script.